CHANGELOG.md

```diff
@@ -80,6 +81,13 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## [v0.65.1]
```

Comment:
We're missing a link for this (and for 2 latest releases):

<img width="911" alt="image" src="https://github.com/user-attachments/assets/2f649bcb-7da6-46fd-ae05-f98945e22f3e" />


Can we fix that in this PR?

Reply:
Will do!

Reply:
Done